games/devtest/mods/lighting/set_lighting.lua

```diff
@@ -0,0 +1,142 @@
+
+local lighting_sections = {
+	{n = "shadows", d = "Shadows",
+		entries = {
+			{ n = "intensity", d = "Shadow Intensity", min = 0, max = 1 }
+		}
+	},
+	{
+		n = "exposure", d = "Exposure",
+		entries = {
+			{n = "luminance_min", d = "Minimum Luminance", min = -10, max = 10},
```

Comment:
Why not use the names as table keys? (Also not a fan of the one-letter field names.)

Reply:
Comes from `/set_lighting`. 
I have changed only filed names and descriptions while being happy that I can reuse existing code to generate interactive formspec.

Reply:
> I have changed only filed names and descriptions

Is there anything stopping this code from being deduplicated?

Reply:
At first time I tried to expand /set_lighting by adding sections. I recognized that the code is not prepared for the management of fields in the table in the table. So I gave up because it was only test command.
The second time I tried to reuse the same functions as /set_lighting uses in the same file. But it gets me into problems because those functions are not designated for reusability. They use global variable/variables. So again, I gave up looking for an easier solution. So I decided to do a separate command for light intensity settings to keep the table structure similar to the table structure, the original command is used to use.
So I do not have to go through developing the formspec generation, which is something I am not good at.

So if somebody refactored the original /set_lighting code to be reusable, I will reuse it.
But I feel no reason to lose time on hard developing of dynamic formspec for testing purposes only.

Reply:
fixed